packages/x-charts-pro/src/ResponsiveChartContainerPro/ResponsiveChartContainerPro.tsx

```diff
@@ -14,13 +18,18 @@ const ResponsiveChartContainerPro = React.forwardRef(function ResponsiveChartCon
   props: ResponsiveChartContainerProProps,
   ref,
 ) {
-  const { width: inWidth, height: inHeight, ...other } = props;
-  const [containerRef, width, height] = useChartContainerDimensions(inWidth, inHeight);
+  const { inHeight, inWidth, chartContainerProps, resizableChartContainerProps } =
+    useResponsiveChartContainerProps(props);
+  const { containerRef, width, height } = useChartContainerDimensions(ref, inWidth, inHeight);
 
   return (
-    <ResizableContainer ref={containerRef} ownerState={{ width: inWidth, height: inHeight }}>
+    <ResizableContainer
+      ref={containerRef}
+      ownerState={{ width: inWidth, height: inHeight }}
+      {...resizableChartContainerProps}
```

Comment:
Seems the owner state could also go into the hook

```suggestion
      {...resizableChartContainerProps}
```